code/modules/unit_tests/outfit_skills.dm

```diff
@@ -0,0 +1,31 @@
+
+/datum/unit_test/outfit_skill_levels/Run()
+	var/max_individual_skill = 4  // Maximum skill level for any single skill
+	var/max_total_skills = 10     // Maximum total skill points across all skills
```

Comment:
10? seriously? 

This should also equip jobs and then check since skills and stats on outfits is going to be phased out and there are already jobs that have no stats or skills on the outfits

Reply:
oh alright then